### PR TITLE
[INLONG-636][Doc] Postgresql cdc suggest setting replica identity to FULL

### LINKED ISSUE #636: [Feature][Doc]Postgres CDC suggest setting replica identity to FULL
### Description

Postgres CDC suggest setting replica identity to FULL, then emitted events for UPDATE and DELETE operations contain the previous values of all columns in the table. See [more].(https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-replica-identity)

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liaorui**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #636

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by yunqingmoswu — COMMENTED** [review]
缺少中文文档

**Comment by dockerzhang** [comment]
@liaorui do we need a Chinese guide doc at the same time for this PR?

**Comment by liaorui** [comment]
> 缺少中文文档

done

**Comment by liaorui** [comment]
> @liaorui do we need a Chinese guide doc at the same time for this PR?

Chinese guide doc has already been completed. Please review again.

**Review by gong — APPROVED** [review]

File: docs/data_node/extract_node/postgresql-cdc.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ build automation tool (such as Maven or SBT) and SQL Client with Sort Connectors
...
-{`<dependency>
+```
+<dependency>
     <groupId>org.apache.inlong</groupId>
     <artifactId>sort-connector-postgres-cdc</artifactId>
     <version>${siteVariables.inLongVersion}</version>
 </dependency>
-`}
-</code></pre>
+```
```
Comment (dockerzhang) [inline docs/data_node/extract_node/postgresql-cdc.md:31]: we need the code to add the version.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/postgresql-cdc.md [hunk 2]
```diff
@@ -116,10 +137,14 @@ TODO: 将在未来支持此功能。
 | debezium.* | 可选 | (none) | String | 将 Debezium 的属性传递给用于从 Postgres 服务器捕获数据更改的 Debezium Embedded Engine。 例如：“debezium.snapshot.mode”=“never”。 查看更多关于 [Debezium 的 Postgres 连接器属性](https://debezium.io/documentation/reference/1.5/connectors/postgresql.html#postgresql-connector-properties)。 |
 | inlong.metric | 可选 | (none) | String | inlong metric 的标签值，该值的构成为groupId&streamId&nodeId。|
 
-**Note**: `slot.name` 建议为不同的表设置以避免潜在的 PSQLException: ERROR: replication slot "flink" is active for PID 974 error。  
-**Note**: PSQLException: ERROR: all replication slots are in use Hint: Free one or increase max_replication_slots. 我们可以通过以下语句删除槽。  
+:::提示
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/postgresql-cdc.md:140]: :::提示
->
:::caution

do not need to translate, it's a fixed template.

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Pull request merged**